DependencyInjection/HttplugExtension.php

```diff
@@ -66,45 +50,47 @@ public function load(array $configs, ContainerBuilder $container)
         }
 
         $this->configurePlugins($container, $config['plugins']);
-        $this->configureClients($container, $config, $toolbar);
-        $this->configureAutoDiscoveryClients($container, $config, $toolbar);
+        $serviceIds = $this->configureClients($container, $config);
+        $autoServiceIds = $this->configureAutoDiscoveryClients($container, $config);
+
+        $toolbar = is_bool($config['toolbar']['enabled']) ? $config['toolbar']['enabled'] : $container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug');
```

Comment:
Can we not directly set the default value of this parameter to %kernel.debug% in the configuration class ?